xbmc/ServiceManager.h

```diff
@@ -153,6 +157,7 @@ class CServiceManager
   std::unique_ptr<CPlatform> m_Platform;
   std::unique_ptr<PLAYLIST::CPlayListPlayer> m_playlistPlayer;
   std::unique_ptr<CSettings> m_settings;
+  std::unique_ptr<KODI::GAME::CControllerManager> m_controllerManager;
```

Comment:
> Anything outside of game should reference this as GAME::CControllerManager

right but ServiceManager is outside of namaspace game. CServiceManager::GetControllerManager vs CServiceManager::GetGameControllerManager

assume some other namespace has a ControllerManager too. I don't know id C++ allows having the same method more than once with just different return types. 
even if it is allowed, it does not look nice